tests/samples/test1.zeek

```diff
@@ -31,8 +31,8 @@ module  Test;
 	  BAR=10,
 	};
 
-	# Another one that we put on one line. That should also remain
-	type SingeLineEnum: enum { FOO, BAR };
+	# Simple enum that used to be a single line, now gets split into multiple.
+	type SimpleEnum: enum { FOO, BAR };
```

Comment:
This test was always a catch-all and hard to debug with the giant changes it could produce, and we hopefully have a better approach with `tests/samples/types.zeek` now. WDYT about moving all enum-related samples in this `export` block into `types.zeek` now and dropping overlapping tests?